packages/code/src/Code.tsx

```diff
@@ -80,6 +94,86 @@ function getWrapperVariantStyle(variant: Variant): string {
   `;
 }
 
+function getSidebarVariantStyle(variant: Variant): string {
+  const colors = variantColors[variant] ?? variantColors[Variant.Light];
+
+  switch (variant) {
+    case Variant.Light:
+      return css`
+        border-color: ${colors[1]};
+        background-color: white;
+      `;
+    case Variant.Dark:
+      return css`
+        border-color: ${colors[1]};
+        background-color: ${colors[1]};
+      `;
+  }
+}
+
+function getCopyButtonStyle(variant: Variant, copied: boolean): string {
```

Comment:
Noticed a problem – The background colors here should all be applied to the `:before` pseudo-element for the button, not the button itself.

Right now, the hover and focus states will cover the button's background color, making the interaction a bit weird.